### PR TITLE
Split symfony components in composer.json

### PR DESCRIPTION
**Pull request by lartist**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Split symfony components in composer.json since symfony/symfony is depracated. Also it might be good to remove unnecessary libraries in another PR
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI 🟢  and auto tests 🟢 https://github.com/lartist/ga.tests.ui.pr/actions/runs/5123726326
| Fixed ticket?     | https://github.com/PrestaShop/PrestaShop/issues/32697
| Related PRs       | 
| Sponsor company   |
Capture d’écran 2023-05-25 à 11 05 22

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -133,13 +133,58 @@
...
+        "symfony/expression-language": "5.4.*",
+        "symfony/filesystem": "5.4.*",
+        "symfony/finder": "5.4.*",
+        "symfony/form": "5.4.*",
+        "symfony/http-client": "5.4.*",
+        "symfony/http-foundation": "5.4.*",
+        "symfony/http-kernel": "5.4.*",
+        "symfony/inflector": "5.4.*",
+        "symfony/intl": "5.4.*",
+        "symfony/ldap": "5.4.*",
```
Comment (FabienPapet) [inline composer.json]: not sure we need this ldap component

Comment (lartist) [reply composer.json]: Hi @FabienPapet, have a look into the description :) . I'm not removing libraries which are not used but just removing a deprecation. I want to leave the same state than before, with symfony/symfony. We should open a second PR and investigate on which library is not used (and I think there is more than just symfony/ldap 👍  )

**Review by jolelievre — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]
🤩

**Comment by kpodemski** [comment]
All auto tests are red.

**Comment by jolelievre** [comment]
https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/5107243555

**Comment by nicosomb** [comment]
Hmm it seems to be timeout for each failing test. Maybe because the installation process takes too long.

**Comment by jolelievre** [comment]
@nicosomb yeah but it's a timeout during the Docker initialization so it appears the install process is failing and maybe the docker is blocked I tried to launch the whole campaign on my fork but it has the same result https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/5107243555

And all the tests are failing, so it's not just a matter of slow environment IMHO, something's happening in the Docker env

**Comment by jolelievre** [comment]
I checked the log archves but it only contains the logs from the Github actions not the internal ones from what's happening inside the docker so we don't have enough feedback right now

**Review by matthieu-rolland — DISMISSED** [review]

**Comment by matthieu-rolland** [comment]
@lartist I just saw that the auto test campaign is all red, I'm putting back the "waiting for author" label (I thought the previous discussions were about the CI)

**Review by jolelievre — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by boherm** [comment]
Waiting for https://github.com/lartist/ga.tests.ui.pr/actions/runs/5123726326

**Comment by jolelievre** [comment]
Thanks @lartist

**Pull request merged**